Gifski/Info.plist

```diff
@@ -23,6 +23,26 @@
 			</array>
 		</dict>
 	</array>
+	<key>NSServices</key>
+	<array>
+		<dict>
+			<key>NSMessage</key>
+			<string>convertToGif</string>
+			<key>NSPortName</key>
+			<string>Gifski</string>
+			<key>NSMenuItem</key>
+			<dict>
+				<key>default</key>
+				<string>Gifski: Convert to GIF</string>
```

Comment:
Any thoughts on whether we should use the above text or `Convert to GIF with Gifski`? I have a lot of services, and I see both formats being used.

Reply:
I just used what I've seen the most in the services I use. I don't have a preference as both are used, so yeah just pick whatever you like and can change it accordingly 😄

Reply:
I think having the app name as prefix is useful if an app exposes multiple services, but when there's only one service, `Convert to GIF with Gifski` reads better, so let's go with that.

Reply:
Good point, gonna update the name as well 👍